.github/workflows/gh-ci.yaml

```diff
@@ -0,0 +1,357 @@
+name: mda_gh_ci
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  MDA_CONDA_MIN_DEPS: "pip pytest mmtf-python biopython networkx cython matplotlib scipy griddataformats hypothesis gsd codecov"
+  MDA_CONDA_EXTRA_DEPS: "seaborn"
+  MDA_CONDA_PY3_DEPS: "netcdf4 scikit-learn joblib>=0.12 clustalw=2.1 chemfiles mock"
+  MDA_CONDA_PY27_DEPS: "clustalw=2.1 netcdf4=1.3.1 six=1.15.0 funcsigs<=1.0.2 mock<=3.0.5"
+  MDA_CONDA_PY35_DEPS: "netcdf4 scikit-learn joblib>=0.12 clustalw=2.1 mock<=3.0.5"
+  MDA_PIP_MIN_DEPS: 'coveralls coverage<5 pytest-cov pytest-xdist'
+  MDA_PIP_EXTRA_DEPS: 'duecredit parmed'
+  MDA_PIP_PY27_DEPS: "setuptools<45.0.0 chemfiles scikit-learn joblib>=0.12"
+  MDA_PIP_PY3_DEPS: ""
+  MDA_PIP_PY35_DEPS: ""
+
+jobs:
+  main_tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        fail-fast: false
+        matrix:
+          os: [ubuntu-latest, ]
+          python-version: [3.6, 3.7, 3.8]
+          extra_deps: ["EXTRA PY3", ]
+          install_hole: [true, ]
+          codecov: [true, ]
+          conda-version: ["latest", ]
+          include:
+            - name: python-2.7
+              os: ubuntu-latest
+              python-version: 2.7
+              install_hole: true
+              codecov: true
+              numpy: numpy=1.16
+              extra_deps: "EXTRA PY27"
+              conda-version: 4.7.10
```

Comment:
Pinning conda didn't really help in resolving dependencies but I found astropy's reason for pinning persuasive (conda is not always so stable from update to update)

Reply:
I think it broke for a 3.7 job?

Reply:
How odd, is this the same version that the ci helpers were pinning to?